frontend/__tests__/inquiry/add_inquiry.test.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/await-thenable */
```

Comment:
I'm confused over the eslint-disable comments. I don't see eslint installed in the project. What am I missing?

Reply:
Codacy
https://github.com/Incredihire/Survey/pull/55/files#annotation_26452229688

Reply:
There's even more after these screenshots!  It's almost every other line.  

[
![codacy3](https://github.com/user-attachments/assets/12c53a31-ca36-4a0c-8758-769a9885caa6)
![codacy2](https://github.com/user-attachments/assets/5c706124-d1b3-4a43-adbe-1d56fb129cc3)
![codacy1](https://github.com/user-attachments/assets/6611f60b-869f-4694-b166-481d53ac6ec4)
](url)

Reply:
https://github.com/Incredihire/Survey/runs/30360405357

Reply:
Codacy!  Here is what the PR looks like with all Codacy es-lint rules enabled
https://github.com/Incredihire/Survey/pull/59/files

Reply:
Could just consider creating a linter spec file for frontend ;)

Reply:
Maybe.  But actually codacy did catch me on some lazy typescript which I did correct.  What's bothersome is that it does not seem to understand the patterns of the libraries we are using. Or something?  I don't know exactly